sys/include/net/ieee802154/radio.h

```diff
@@ -588,14 +588,14 @@ struct ieee802154_radio_ops {
      *
      * @param[in] dev IEEE802.15.4 device descriptor
      * @param[in] short_addr the IEEE802.15.4 short address
-     * @param[in] ext_addr the IEEE802.15.4 extended address
+     * @param[in] ext_addr the IEEE802.15.4 extended address (Network Endian)
```

Comment:
```suggestion
     * @param[in] ext_addr the IEEE802.15.4 extended address (Network Byte Order)
```
is the correct term ;-)

Reply:
tadaaaa :tada:

https://github.com/RIOT-OS/RIOT/pull/14371/commits/e76f72039472811b74b19b796c64a1e26cd6357a